examples/src/retrievers/parent_document_retriever_rerank.ts

```diff
@@ -0,0 +1,66 @@
+import { OpenAIEmbeddings } from "@langchain/openai";
+import { CohereRerank } from "@langchain/cohere";
```

Comment:
So this doesn't yet implement `BaseDocumentCompressor` - I will fix that before merging this.

Reply:
Thank you

Reply:
Hi, I've changed a code so you can feed any document compressor to both Parent Retreiver and Multi Query Retreiver so it's much cleaner and it's up to developer what to use.